Rework install command

mbround18 opened the pull request:
- Closes https://github.com/mbround18/valheim-docker/issues/1229

kojinb commented:
hey, is the `USE_PUBLIC_BETA` arg necessary? and if it is, should it be in the `README.md`?

mbround18 commented:
Yes, thats how I track whether you use beta or not

mbround18 commented:
@kojinb apologies that image had some issues and I was in a rush,  I just fixed it up and reworked it over at: 

```
  mbround18/valheim:sha-d7de53f
```

kojinb commented:
hey @mbround18, no worries man! hope you enjoyed your camping trip

I pulled your latest image using the sha you gave me, rebuilt the container and everything seems to work correctly! it pulled the beta branch version as expected, and I also connected successfully with my client ✔️
